playbooks/roles/airship-deploy-tempest/defaults/main.yml

```diff
@@ -22,3 +22,5 @@ cirros_test_image_url: "http://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_
 use_blacklist: true
 # Flag to hide tempest pod logs during ansible tempest run. Default is True (show tempest logs)
 print_tempest_log: "{{ lookup('env','AIRSHIP_TEMPEST_LOG_STDOUT') | default('True', true) | bool }}"
+# wheter to output tempest in subunit format
+tempest_subunit: "{{ lookup('env', 'SOCOK8S_TEMPEST_PARSE_CI') | default(False, True) | bool }}"
```

Comment:
Why do we want it false by default? That will mean you'll need to explicitly set it before the CI run to get the tempest results...

Reply:
So we can set it to true on the jenkinsfile but on developers machines is not run by default :D

Reply:
Ah, true, I'm complaining at the wrong place...